### DISCUSSION TITLE
amamba/quickstart/argocd-rollout

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2281] [Announcements]
# amamba/quickstart/argocd-rollout

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/amamba/quickstart/argocd-rollout.html?h=gateway

**Comment by wenting-guo** [comment]
创建应用时候，不需要“开启灰度发布”，产品已经更新了
